### ISSUE TITLE
error Couldn't find the binary git

### ISSUE THREAD

**Issue by emiliokyp** [stefanoeb/jest-action#3]
main.yml:
```# This is a basic workflow to help you get started with Actions

name: Tests

# Controls when the action will run. Triggers the workflow on push or pull request 
# events but only for the master branch
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
    
jobs:
  test:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1
    - name: Run Jest
      uses: stefanoeb/jest-action@1.0.2
```

log:
```Run stefanoeb/jest-action@1.0.2
  with:
/usr/bin/docker run --name e87b52a49f9c02c23a45f889d4b47f4d6f2521_ae8242 --label e87b52 --workdir /github/workspace --rm -e INPUT_JESTARGS -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/salestribe-app/salestribe-app":"/github/workspace" e87b52:a49f9c02c23a45f889d4b47f4d6f2521  ""
## Your environment is not ready yet. Installing modules...
## Running Jest
error Couldn't find the binary git
```

Not sure if this is related to your action or something I've setup wrong. Any help would be appreciated.

**Comment by stefanoeb** [comment]
Hey @emiliokyp 

I've tried using your `main.yml` on a dummy project and it worked ok. You can check it here: https://github.com/stefanoeb/github-actions-app/pull/12

However, it seems like it's trying to use a git binary where it should be only installing modules. Out of curiosity, does any of your modules are installed via `git`? If so, are they private (using ssh?)

Thanks for reporting this

**Comment by eduter** [comment]
I'm having the same problem and, yes, one of my dependencies is installed from a public GitHub repo, not from npm, so that makes sense. Is there a solution for this case?

**Comment by stefanoeb** [comment]
@eduter @emiliokyp I've added git + openssh to the base docker image on this action.

This is already available under the 1.0.3 version (update your workflow to use `uses: stefanoeb/jest-action@1.0.3`)
